proto/stream_plan.proto

```diff
@@ -306,6 +313,7 @@ message HashJoinNode {
   // Whether to optimize for append only stream.
   // It is true when the input is append-only
   bool is_append_only = 14;
+  BandJoinCondition band_condition = 15;
```

Comment:
We have another field `inequality_pairs` which has similar functionality to `band_condition`. Is it possible to merge them?

Reply:
> We have another field `inequality_pairs` which has similar functionality to `band_condition`. Is it possible to merge them?

too hard for me 😭

Reply:
😭 Will take a look in detail later.